package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Simple plugin for Leaflet that adds fullscreen button to your maps.",
   "main": "Control.FullScreen.js",
   "scripts": {
-    "lint": "eslint --config .eslintrc Control.FullScreen.js",
+    "lint": "npx eslint . && stylelint **/*.css && prettier --check .",
+    "lint:fix": "npx eslint --fix . && stylelint --fix **/*.css && prettier --write .",
```

Comment:
same here - I apologize for the trouble I created.

Reply:
Same here: no trouble! 🙂 I removed npx again.